server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -267,7 +267,7 @@
     @Deprecated
     public static final Setting<Boolean> INDEX_PLUGINS_REPLICATION_TRANSLOG_RETENTION_LEASE_PRUNING_ENABLED_SETTING =
         Setting.boolSetting("index.plugins.replication.translog.retention_lease.pruning.enabled", false,
-            Property.IndexScope, Property.Dynamic, Property.Deprecated);
```

Comment:
We want this deprecated in 1.1 because we intend to refactor it to the ccr plugin in 1.2.